### ISSUE TITLE
Notify curators that they will be added to a study's "Submitted by" property

### ISSUE THREAD

**Issue by jimallman** [OpenTreeOfLife/opentree#620]
We've had some user confusion about whether their work (eg, applying a CC-0 waiver) in an existing study will be attributed to the original submitter. We might want to add a notice that anyone saving changes to a study will have their name added after saving it. Here are a couple of visual treatments:

![screen shot 2015-03-18 at 3 09 24 pm](https://cloud.githubusercontent.com/assets/446375/6717197/49a3b0fa-cd81-11e4-9346-6ca0c1327070.png)

![screen shot 2015-03-18 at 3 10 17 pm](https://cloud.githubusercontent.com/assets/446375/6717198/52dc99d4-cd81-11e4-822c-6b3f44c7beee.png)

On a related note, we should also make the new curator name appear *immediately* after saving, even if the curator keeps working--just as we update the History tab.

**Comment by kcranston** [comment]
Using the application of the waiver as an example seems to role two issues into one here: 1. whether the waiver / license is associated with a study or a curator; 2. how to properly attribute curation work on a study. Can you clarify what the confusion was about?

**Comment by jimallman** [comment]
I believe it's a combination of both. A curator had reported (via email to @josephwb):

> I was not able to create a separate file as this already had a file under “O’Leary” and uploaded by Brown. So when I tried to select the “uploaded by author” and therefore, choose “CC-0”, it appeared that this was all being done under “Brown", which seemed rather confusing, especially for an outsider who may come across the data here on open tree. Not sure if that is a bug, or just a feature to-be-developed.

I gather that by "create a separate file", they started to submit the same study and realized that it was already present in the system, attributed to another submitter (Brown). So it seems there was a concern that any changes made by another curator would be attributed to Brown.

**Comment by jar398** [comment]
On Wed, Mar 18, 2015 at 3:25 PM, Jim Allman <notifications@github.com>
wrote:

> I believe it's a combination of both. A curator had reported (via email to
> @josephwb <https://github.com/josephwb>):
>
> I was not able to create a separate file as this already had a file under
> “O’Leary” and uploaded by Brown. So when I tried to select the “uploaded by
> author” and therefore, choose “CC-0”, it appeared that this was all being
> done under “Brown",
>
>
How did the curator become aware of this - how did it "appear that this was
being done under 'Brown'"? Was it because of a DOI lookup alert? - but that
would have come before the CC0 checkbox, yes? I thought that ordinarily,
duplicates were allowed, because they were not detected. Why were they "not
able to create a separate file"? If they got to the point of checking CC0,
they had already started to create a new study (object), so how could they
know at that point anything about Brown? Or was it that they did a search,
found the study, and then *decided* not to create a separate "file"?

> which seemed rather confusing, especially for an outsider who may come
> across the data here on open tree. Not sure if that is a bug, or just a
> feature to-be-developed.
>
> I gather that by "create a separate file", they started to submit the same
> study and realized that it was already present in the system, attributed to
> another submitter (Brown). So it seems there was a concern that any changes
> made by another curator would be attributed to Brown.
>
I don't understand exactly what the concern was. Did they take it on
themselves to avoid doing things that could be attributed to Brown, because
Brown might not like that, or were they concerned that they were going to
put in effort that would be misattributed to Brown, and thereby be deprived
of credit that they were due? - Maybe these are the same. Perhaps this
person formed an ownership model in their head, and then committed
themselves to enforcing it - the "not able" came not from the UI but from
their own mind. In this case we need to make clear in the UI that there is
no ownership. The details matter because they dictate *where* in the UI we
should intervene.

(This relates to my request that the original curator, not the latest one,
be shown in the study list. (I have entered a bunch of DOIs and that causes
me to look like 'the' study curator, which I'm not qualified to be.))

**Comment by jimallman** [comment]
> Or was it that they did a search, found the study, and then *decided* not to create a separate "file"?

That's my understanding, yes. @josephwb, does this sound correct?

> Perhaps this person formed an ownership model in their head, and then committed
themselves to enforcing it - the "not able" came not from the UI but from their own mind. 

I suspect this is the case -- that they felt the original submitter has "dibs" on this study. The mockups above try to explain that the listed curator(s) are simply those who've contributed to the study **so far**, but I see your point about trying to really explain our no-ownership model.

Regarding which curator(s) appear in the study list, this is due to a [known issue in **oti**](https://github.com/OpenTreeOfLife/oti/issues/35), which doesn't index multiple values where needed (eg, curators and tags).

**Comment by jar398** [comment]
The mockups look like the right kind of thing, and they might do the trick.
This would go on the metadata tab, right?

My feeling is that only one person can 'submit' a study. The curators would
be the submitter and all others who made improvements. Something like
'Additional curation by: (your name here)' although that's too flip.

Yes, I know about the known issue, I remember talking about it and your
diagnosis. Sorry to bring it up again.

**Comment by jimallman** [comment]
> This would go on the metadata tab, right?

Yes. And I see your point about the labels used here. Maybe we should ditch 'Submitted by' (which feels like a loaded term implying ownership/authorship) in favor of 'Curated by:', and keep the current behavior; i.e., on initial submission of a study, the submitter becomes the first-listed curator. Thoughts?
